frontend/kobweb-compose/src/jsMain/kotlin/com/varabyte/kobweb/compose/ui/modifiers/TextModifiers.kt

```diff
@@ -67,3 +67,15 @@ fun Modifier.wordBreak(wordBreak: WordBreak): Modifier = styleModifier {
 fun Modifier.writingMode(writingMode: WritingMode) = styleModifier {
     writingMode(writingMode)
 }
+
+fun Modifier.rubyPosition(rubyPosition: RubyPosition) = styleModifier {
+    rubyPosition(rubyPosition)
+}
+
+fun Modifier.wordSpacing(wordSpacing: WordSpacing) = styleModifier {
+    wordSpacing(wordSpacing)
+}
+
+fun Modifier.wordSpacing(length: CSSLengthNumericValue) = styleModifier {
+    wordSpacing(WordSpacing.of(length))
```

Comment:
```suggestion
fun Modifier.wordSpacing(value: CSSLengthNumericValue) = styleModifier {
    wordSpacing(WordSpacing.of(value))
```
For consistency with the name change I suggested for the `of()` function.